### DISCUSSION TITLE
Live painting with openpose and Layered Diffusion

### DISCUSSION THREAD

**Discussion by Iter8tion** [lllyasviel/stable-diffusion-webui-forge#533] [General]
Question: aside from Krita with Krita ai diffusion plugin (separate program from running internally to SD webui), does anyone know if there is an extension built or being built that will allow real-time image generation via Openpose....Combined with the new SD LayeredDiffusion (all within Forge or A1111)?  With the ability to continually adjust the openpose model above a separate background image?  This way, you have an established background and then have the ability to adjust the character position, blending to the background, without actually having to have the result combined (they remain separate entities until you find an image combination you like).  At that point, you are left with 3 images: the background, the character, and the combined version.

**Comment by a3nima** [comment]
you can already load an openpose json and edit it to your liking inside forge and use layer diffusion with that controlnet, maybe I dont get what your're exactly after?  :D

**Reply by Iter8tion** [reply]
Hi. Yea, I understand that process. I am looking for an even better/more interactive process. One where you can directly change the character on the fly over the desired background and have it update right away as you change the character. Rather than go into openpose, load background, adjust rig, then send back over to controlnet, to then render. Or take into layerdiffusion and render.  There are a few extra steps, you see? And a lot less on the fly...like live diffusion.  Am I making sense or my description?

**Reply by a3nima** [reply]
the pose edit function could have more features incl. background function, that would be helpful, of course a combined method is better. there was an extension that would work that way..

in my experience another extension that combine different things (openpose+layer diffusion)are more realistic than the GUI makers building something like that into their Webui, but we'll see, would be nice of course :)
